internal/ethereum/prepare_transaction.go

```diff
@@ -94,7 +128,60 @@ func (c *ethConnector) prepareCallData(ctx context.Context, req *ffcapi.Transact
 	}
 
 	return callData, method, err
+}
+
+func (c *ethConnector) prepareDeployData(ctx context.Context, req *ffcapi.ContractDeployInput) ([]byte, *abi.Entry, error) {
+	// Parse the bytecode as a hex string, or fallback to Base64
+	bytecode, err := hex.DecodeString(strings.TrimPrefix(req.Bytecode.String(), "0x"))
```

Comment:
I think for a `JSONAny` this will have `"0x1234` rather than `0x1234`.

Instead it'll be necessary to do:
```go
var bytecodeString string
err := json.Unmarshal(req.Bytecode.Bytes(), &bytecodeString)
if err != nil {
    return nil, nil, i18n.NewError(ctx, msgs.MsgDecodeBytecodeFailed)
}
```

Reply:
I'll merge this PR though - and work forward from there

Reply:
I think you are correct. I didn't actually test that last minute change. I guess that's the argument against changing code right before walking out the door.